corehq/apps/app_manager/xpath_validator/js/xpathValidator.js

```diff
@@ -0,0 +1,11 @@
+global.SchemeNumber = require('js-xpath/lib/schemeNumber').SchemeNumber;
```

Comment:
Yes, this should be changed to something like `require('schemeNumber')`, making `schemeNumber` a requirement of this xpath validator (added to package.json). As is the requirement uses an implementation detail of the `js-xpath` library. If a future version of `js-xpath` does not use `schemeNumber` it will break this require call.

Also, assigning to a global is a red flag, but I don't have enough context to suggest a proper fix.